client/app/components/land-use-action.js

```diff
@@ -99,11 +153,6 @@ export default class LandUseActionComponent extends Component {
       if (includesA && !includesB) {
         return -1;
       }
-
-      // if top action is from db and bottom action is new -- reorder
-      if (!includesA && includesB) {
```

Comment:
why the removal of this?

Reply:
i'm trying to understand why codecov is saying this line isn't hit.

i pushed this change to see if codecov was less mad, it seems it is...

just trying to understand why codecov is red, not gonna merge.

Reply:
@trbmcginnis any theories on why removing these lines actually increases coverage? i don't think they're logically ever hit? is there a case in which they would be hit?

Reply:
@trbmcginnis i think because we use `unshiftObject` the new item is never actually at the bottom of the list?

Reply:
Oh interesting -- I'm thinking this would be hit in the case of an action that is alphabetically situated higher (e.g. `Change in City Map`) and exists in the original db being compared to an action that is alphabetically situated lower (e.g. `Landfill`) but added by the user and therefore should be sorted ABOVE `Change in City Map`. I'm wondering though if these are sorted correctly by the other conditions before it hits this condition and therefore it's not necessary? idk, I can look into it and try to update tests to include more cases